rbac/server/api/roles.py

```diff
@@ -80,6 +84,34 @@
         await utils.send(
             request.app.config.VAL_CONN, batch_list, request.app.config.TIMEOUT
         )
+        conn.close()
+        if role_title != "NextAdmins":
+            distinguished_name_formatted = "CN=" + role_title + "," + GROUP_BASE_DN
+            data_formatted = {
+                "created_date": r.now(),
+                "distinguished_name": distinguished_name_formatted,
+                "group_nickname": role_title,
+                "group_types": -2147483646,
+                "name": role_title,
+                "remote_id": distinguished_name_formatted,
+            }
+            outbound_entry = {
+                "data": data_formatted,
+                "data_type": "group",
+                "sync_type": "delta",
```

Comment:
Outbound entries does not need a sync_type field, but the other fields are required for outbound queue processing.

Reply:
removed